server/cli/init/base-file-sources/hooks.js

```diff
@@ -1,44 +1,50 @@
 'use strict';
 
-/* eslint-disable no-var, prefer-arrow-callback */
+/* eslint-disable no-var*/
 var constants = require('../../../constants')
 var cucumber = require('cucumber');
 var cucumberHtmlReport = require('cucumber-html-report');
 var log = require('npmlog');
 var moment = require('moment');
 var path = require('path');
 var Promise = require('bluebird');
-
 var fs = Promise.promisifyAll(require('fs'));
- /* eslint-disable new-cap */
-var jsonFormatter = cucumber.Listener.JsonFormatter();
-var outputDir = path.join(__dirname, '../', constants.REPORT_DIR)
 
 
-function createFileName (file, extension) {
-    /* eslint-disable prefer-template */
-    return file + moment().format('YYYY-MM-DD_HH-mm-ss') + '.' + extension;
-}
+function createReporter () {
+    /* eslint-disable new-cap */
+    var jsonFormatter = cucumber.Listener.JsonFormatter();
+    var outputDir = path.join(__dirname, '../', constants.REPORT_DIR)
+    jsonFormatter.log = content => jsonReportWriter(content);
```

Comment:
Nah sorry, I meant as in Function.prototype.bind. 

`jsonFormatter.log = content => jsonReportWriter(content);` is equivalent to `jsonFormatter.log = jsonReportWriter;` here because we don't use `this`.